projects/addon-editor/components/editor-new/editor-new.component.ts

```diff
@@ -102,6 +106,16 @@ export class TuiEditorNewComponent
         );
     }
 
+    writeValue(value: string | null): void {
+        const processed = tuiIsPresent(value) ? this.contentProcessor(value) : value;
```

Comment:
Maybe just `this.contentProcessor(value || '')` for simplicity?

Reply:
fixed